src/yunohost/certificate.py

```diff
@@ -567,7 +569,11 @@ def _fetch_and_enable_new_certificate(domain, staging=False):
         raise MoulinetteError(errno.EINVAL, m18n.n(
             'certmanager_cert_signing_failed'))
 
-    intermediate_certificate = requests.get(INTERMEDIATE_CERTIFICATE_URL).text
+    try:
+        intermediate_certificate = requests.get(INTERMEDIATE_CERTIFICATE_URL, timeout=30).text
+    except Timeout:
+        # XXX what should we do here? retry?
```

Comment:
If we didn't get the intermediate certificate, the certificate won't work, so I'd say we just do a `raise MoulinetteError(errno.EINVAL, m18n.n('certmanager_couldnt_fetch_intermediate_cert'))`

(Btw, I never know if we do care about which errno to use ?)